### PR TITLE
Improve base-conversion

### PR DESCRIPTION
**Pull request by sourabhv**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/numbers/base-conversion.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/numbers/base-conversion.md?ref=06dbe6a488c75af3012263d087cbd365c9db22be)

File: javascript/core/numbers/base-conversion.md [hunk 1]
```diff
@@ -31,38 +31,55 @@ aspects:
...
 
-Base conversion for `int` data types is easily achievable with the help of `toString` and `parseInt` functions. 
+Base conversion for numerical values in JavaScript is easily achievable
+with the help of `toString` and `parseInt` functions.
 
-Convert decimal to hexadecimal:
+
+### toString
+
+`<number>.toString(base)` operates on a number and converts it into a string in given base
```
Comment (kapnobatai137) [inline javascript/core/numbers/base-conversion.md]:
```suggestion
`<number>.toString(base)` operates on a number and converts it into a string in the given base.
```

File: javascript/core/numbers/base-conversion.md [hunk 2]
```diff
@@ -31,38 +31,55 @@ aspects:
...
 ```javascript
-(int).toString(16);
+let number = 123
+number.toString(16) // "7b"
+number.toString(8) // "173"
+number.toString(2) // "1111011"
 ```
 
-Convert decimal to octal:
+Note that the type of returned valye is string
```
Comment (kapnobatai137) [inline javascript/core/numbers/base-conversion.md]:
```suggestion
Note that the type of returned value is a string.
```

File: javascript/core/numbers/base-conversion.md [hunk 3]
```diff
@@ -31,38 +31,55 @@ aspects:
...
+let number = 123
+typeof number.toString(16) // "string"
 ```
-Vice-versa convert hexadecimal to decimal:
+
+### parseInt
+
+`parseInt(<string>, <base>)` takes a string and it's base and converts it into a decimal number.
+
+For example, parsing a hexadecimal, octal and binary number in string form into a decimal
```
Comment (kapnobatai137) [inline javascript/core/numbers/base-conversion.md]:
```suggestion
For example, parsing a hexadecimal, octal and binary number in string form into a decimal:
```

File: javascript/core/numbers/base-conversion.md [hunk 4]
```diff
@@ -31,38 +31,55 @@ aspects:
...
+parseInt("7b", 16) // 123
+parseInt("173", 8) // 123
+parseInt("1111011", 2) // 123
 ```
-Convert octal to decimal:
-```javascript
-parseInt(string,8);
 
-```
+The type of returned value is `number`
```
Comment (kapnobatai137) [inline javascript/core/numbers/base-conversion.md]:
```suggestion
The type of returned value is `number`.
```

**Pull request merged**